### ISSUE TITLE
co-exist with lsp-backend texlab

### ISSUE THREAD

**Issue by epichub** [emacs-grammarly/lsp-grammarly#13]
It seems spacemacs has changed the default lsp backend of latex-mode to texlab. Which is nice I guess. But this seems to have squeezed out lsp-grammarly. I have investigated lsp-mode and it seems it should support multiple backends (https://github.com/emacs-lsp/lsp-mode/issues/424). I cannot figure out why lsp-grammarly does not load, only texlab. I use the default config provided by the README.md of this project.

**Comment by jcs090218** [comment]
I think the `:add-on?` flag is required? Should fix in 57a698e69c0dacbcad30398c262cdc5711efba71.

Can you upgrade to the latest version and see if this work for you? Thanks!

**Comment by epichub** [comment]
That worked. Wow that was fast response. I have your git cloned so just evaluated the new register code.
BTW, maybe lsp-grammarly can be installed from git via quelpa use-package (https://github.com/quelpa/quelpa-use-package)?
I have this
```
(use-package keytar
      :quelpa (keytar :fetcher github :repo "emacs-grammarly/keytar"))
(use-package lsp-grammarly
      :quelpa (lsp-grammarly :fetcher github :repo "emacs-grammarly/lsp-grammarly")
      :ensure t
      :hook (text-mode . (lambda ()
                                   (require 'lsp-grammarly)
                                   (lsp-deferred))))
```

Which also seems to work fine.
